### ISSUE TITLE
Division by zero in _list.html.twig

### ISSUE THREAD

**Issue by ausi** [contao/contao#5081]
### Affected version(s)

5.0.0-RC1

### Description

`items_per_page` might be zero or null which causes a `Division by zero` error.

https://github.com/contao/contao/blob/eb669536225740719a99927f1518605e4bfe2c48/core-bundle/contao/templates/_new/component/_list.html.twig#L50

**Comment by ausi** [comment]
```twig
{% set number_of_pages = items_per_page ? ((limit_elements ?: number_of_items) / items_per_page)|round(0, 'ceil') : 1 %}
```
This the correct fix?

**Comment by m-vo** [comment]
LGTM 👍

**Comment by ausi** [comment]
Closed in Favor of #5140
